### PR TITLE
[FLINK-6687] [web] Activate strict checkstyle for flink-runtime-web

### PR DESCRIPTION
**Pull request by zentol**
Lot's of missing javadocs, trailing spaces in javadocs, trailing tabs and missing dots.

Very little code is affected.

### CODE REVIEW

File: flink-runtime-web/src/test/java/org/apache/flink/runtime/webmonitor/handlers/checkpoints/CheckpointStatsSubtaskDetailsHandlerTest.java [hunk 1]
```diff
@@ -57,6 +57,9 @@
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
+/**
+ * Tests for the CheckpointStatsSubtaskDetailsHandlerTest.
```
Comment (greghogan) [inline flink-runtime-web/src/test/java/org/apache/flink/runtime/webmonitor/handlers/checkpoints/CheckpointStatsSubtaskDetailsHandlerTest.java]: Self-reference

File: flink-runtime-web/src/test/java/org/apache/flink/runtime/webmonitor/metrics/TaskManagerMetricsHandlerTest.java [hunk 2]
```diff
@@ -33,6 +34,9 @@
 import static org.junit.Assert.assertNull;
 import static org.powermock.api.mockito.PowerMockito.mock;
 
+/**
+ * Tests for the TaskManagersMetricsHandler.
```
Comment (greghogan) [inline flink-runtime-web/src/test/java/org/apache/flink/runtime/webmonitor/metrics/TaskManagerMetricsHandlerTest.java]: `Manager`

File: flink-runtime-web/src/main/java/org/apache/flink/runtime/webmonitor/handlers/RequestHandler.java [hunk 3]
```diff
@@ -36,13 +37,15 @@
 	 * respond with a full http response, including content-type, content-length, etc.
 	 *
 	 * <p>Exceptions may be throws and will be handled.
-	 * 
+	 *
+*
```
Comment (zentol) [inline flink-runtime-web/src/main/java/org/apache/flink/runtime/webmonitor/handlers/RequestHandler.java]: remnants of a regex replacement gone wrong :/

File: flink-runtime-web/src/main/java/org/apache/flink/runtime/webmonitor/RuntimeMonitorHandler.java [hunk 4]
```diff
@@ -29,15 +34,8 @@
...
 import io.netty.handler.codec.http.router.Routed;
-
-import org.apache.flink.configuration.ConfigConstants;
-import org.apache.flink.runtime.instance.ActorGateway;
-import org.apache.flink.runtime.webmonitor.handlers.RequestHandler;
-import org.apache.flink.util.ExceptionUtils;
-
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-
```
Comment (zentol) [inline flink-runtime-web/src/main/java/org/apache/flink/runtime/webmonitor/RuntimeMonitorHandler.java]: Agreed.

**Comment by zentol** [comment]
@greghogan I've added a separate import block for scala and addressed all comments.

**Comment by greghogan** [comment]
+1

**Comment by zentol** [comment]
I'll merge this after the metrics/python/annotations checkstyle PR's are merged. Need to rebase due to the newly added rule.

**Pull request closed without merging**